packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -50,11 +50,24 @@ const AppConfig = ({children, locals = {}}) => {
     const headers = {
         'correlation-id': correlationId
     }
+    const onClient = typeof window !== 'undefined'
 
     const commerceApiConfig = locals.appConfig.commerceAPI
 
     const appOrigin = getAppOrigin()
 
+    const isLocalhost = appOrigin.includes('localhost')
```

Comment:
Do we need to be concerned about ip values for local host? like `127.0.0.1` for macs? If so we might want to implement this in an sdk somewhere so we don't over complicate this component.

Reply:
code removed